rsts/community/contribute.rst

```diff
@@ -0,0 +1,255 @@
+######################
+Contributing to Flyte
+######################
+
+Thank you for taking the time to contribute to Flyte! Here are some guidelines for you to follow, which will make your first and follow-up contributions easier.
+
+.. note::
+    Please read our `Code of Conduct <https://lfprojects.org/policies/code-of-conduct/>`_ before contributing to Flyte.
+
+Code
+====
+An issue tagged with ``good first issue`` is the best place to start for first-time contributors. Look into them `here <https://github.com/flyteorg/flyte/labels/good%20first%20issue>`_.
+
+To take a step ahead, check out the repositories available under `flyteorg <https://github.com/flyteorg>`_.
+
+**Appetizer (for every repo): Fork and clone the concerned repository. Create a new branch on your fork and make the required changes. Create a pull request once your work is ready for review.** 
+
+.. note::
+    Note: To open a pull request, follow this `guide <https://guides.github.com/activities/forking/>`_.
+
+*A piece of good news -- You can be added as a committer to any ``flyteorg`` repo as you become more involved with the project.*
+
+Example PR for your reference: `GitHub PR <https://github.com/flyteorg/flytepropeller/pull/242>`_. A couple of checks are introduced to help in maintaining the robustness of the project. 
+
+#. To get through DCO, sign off on every commit. (`Reference <https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md>`_) 
+#. To improve code coverage, write unit tests to test your code.
+
+.. note::
+    Format your Go code using the ``go fmt`` command and Python code with ``black`` (using ``make fmt`` command which contains both black and isort). 
```

Comment:
we actually use `go imports`.
We run `golang-ci` for linting and  it follows goimports formating
https://github.com/flyteorg/flytepropeller/blob/master/boilerplate/lyft/golang_test_targets/Makefile#L11-L13